src/stylesheets/stats.scss

```diff
@@ -52,34 +52,34 @@
 
   .list-view {
     .stats-list {
-      // pointer-events: none;
+      pointer-events: none;
       width: 66%;
     }
   }
 
-  .adobe-data, .yearly-data {
-    > ul {
+  .adobe-data, .yearly-data, .monthly-data {
+    > ul, .disable-events {
       pointer-events: none;
       margin: 0;
       padding: 10px 0;
       list-style: none;
       width: 60%;
-      .header-bar {
-        display: flex;
-        justify-content: space-between;
-        align-items: center;
-        padding: 10px 20px;
-        background: $blue-tint;
-        border-bottom: 1px solid $blue-dark;
-        border: 1px solid $blue-dark;
-        margin-top: 5px;
-      }
-      .list-holder {
-        border-left: 1px solid $blue-dark;
-        border-right: 1px solid $blue-dark;
-        border-bottom: 1px solid $blue-dark;
-        padding: 10px 20px;
-      }
+    }
+    .header-bar {
```

Comment:
Why were these rules moved out?

Reply:
The `.header-bar` thing in `MonthlyDataTab` isn't inside a `ul`.